docs/guides/heroku.md

```diff
@@ -1,9 +1,132 @@
 <!--[meta]
 section: guides
-title: Deploying Keystone to Heroku
-draft: true
+title: Heroku
+subSection: deployment
 [meta]-->
 
 # Deploying Keystone to Heroku
 
-In this guide we will learn how to deploy our Keystone system to Heroku.
+This quick start guide shows how to deploy your first KeystoneJS app to Heroku and use MongoDB Atlas as our database service.
+
+## Requirements
+
+Before we start, make sure you have:
+
+- [Git](https://git-scm.com/downloads) installed with `git --version`.
+- [Docker](https://docs.docker.com/install/) installed with `docker -v`.
+
+And that you have created your first KeystoneJS app:
+
+- If not, see the guide [Your first KeystoneJS app in 5 minutes](https://www.keystonejs.com/quick-start/).
+
+At the moment it is only possible to deploy the TODO starter project without any modifications. Deploying [the STARTER starter project](#deploy-the-starter-starter-project) or [the BLANK starter project](#deploy-the-blank-starter-project) requires a few tweaks.
+
+## MongoDB Atlas
+
+MongoDB Atlas has a free tire that we can use as our database service. You need to create an account, but you can use the free tire.
```

Comment:
All these should be "trial" nor "tire".

Reply:
Pretty sure "free tier" is a valid term

Reply:
Yes, but it says "tire" (as in wheel) not "tier".

Reply:
Same same for a Swed 😄. I think the Atlas service is called tier and Heruko trial. I'll check and update.